wp-config-sample.php

```diff
@@ -58,15 +58,15 @@
 /**#@-*/
 
 /**
- * WordPress Database Table prefix.
+ * ClassicPress Database Table prefix.
  *
  * You can have multiple installations in one database if you give each
  * a unique prefix. Only numbers, letters, and underscores please!
  */
 $table_prefix  = 'wp_';

```

Comment:
AFAIK table_prefix can be anything, so would perhaps be nice if we make the default one `cp_`?